src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/PrisonApiService.java

```diff
@@ -212,5 +215,20 @@ public PrisonAppointment getPrisonAppointment(long appointmentId) {
                 .blockOptional()
                 .orElse(null);
     }
+
+    public void deleteAppointment(final Long appointmentId) {
+        try {
+            webClient.delete()
+                    .uri("/appointments/{appointmentId}", appointmentId)
+                    .retrieve()
+                    .bodyToMono(String.class)
```

Comment:
The DELETE response doesn't have a body.

Reply:
this is how all the other void methods are consumed. Not too sure how to handle this otherwise?

Reply:
ahh `.toBodilessEntity().block()`

Reply:
updated with   .toBodilessEntity().block()